server/apiCalls/transformers/analyses.test.js

```diff
@@ -32,10 +38,16 @@ const offering_SF1625_202122 = {
   courseCode: 'SF1625',
   offeringId: '2',
   period: 'P1',
+  courseRoundApplications: [
+    {
+      course_round_application_code: '2',
+    },
+  ],
 }
 const analysis_SF1625_202122_base = (courseCode = 'SF1625') => ({
   courseCode,
-  roundIdList: '2',
+  roundList: '2',
```

Comment:
`roundList ` has to be removed to be sure tests working properly

Reply:
Removed